app/src/main/java/com/omniimpact/javaexperiments/Activities/ActivityMain.java

```diff
@@ -10,14 +10,39 @@
 
 import Fragments.FragmentMain;
 
+/**
+ * Main activity for the application where we start execution.
+ *
+ * Activities are individual "whole windows" of a Android application.
+ */
 public class ActivityMain extends AppCompatActivity {
 
+    /**
+     * Occurs on the instantiation of the application, creates the initial
+     * layout and sets it to the current content view.
+     * TODO: Add a point about how this is different from onStart()
+     * @param savedInstanceState TODO: Unknown, what is this?
```

Comment:
if you want to explicitly pass stateful information back to the activity if it is otherwise cleared from memory, you can return a Bundle from onSaveInstanceState() that will be passed back to onCreate() to remember where you left off.

Reply:
Added to comments.